input/system/crunchy_bridge/client.go

```diff
@@ -0,0 +1,258 @@
+package crunchy_bridge
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/pganalyze/collector/util"
+)
+
+const apiBaseURL = "https://api.crunchybridge.com"
+
+type Client struct {
+	http.Client
+
+	BaseURL     string
+	BearerToken string
+	ClusterID   string
+}
+
+type ClusterInfo struct {
+	CPU        int32   `json:"cpu"`
+	CreatedAt  string  `json:"created_at"`
+	Memory     float32 `json:"memory"`
+	Name       string  `json:"name"`
+	PlanID     string  `json:"plan_id"`
+	ProviderID string  `json:"provider_id"`
+	RegionID   string  `json:"region_id"`
+	Storage    int32   `json:"storage"`
+}
+
+type MetricViews struct {
+	Name   string         `json:"name"`
+	Series []MetricSeries `json:"series"`
+}
+
+type MetricSeries struct {
+	IsEmpty bool          `json:"is_empty"`
+	Name    string        `json:"name"`
+	Points  []MetricPoint `json:"points"`
+	Title   string        `json:"title"`
+}
+
+type MetricPoint struct {
+	Time  string  `json:"time"`
+	Value float64 `json:"value"`
+}
+
+type CPUMetrics struct {
```

Comment:
[note for reviewer] Technically we don't need types (as well as corresponding functions) from CPUMetrics to LoadAverageMetrics as we use gopsutil for that, not API.
I initially implemented it assuming that we had to use the API for them as we were not able to get correct metrics of them using gopsutil, but I was wrong and we can actually get accurate metrics using gopsutil.
I could simply delete them but I figured that it won't hurt to have them here.